src/Plugin.cc

```diff
@@ -195,8 +198,8 @@ std::pair<bool, Val*> Plugin::HookCallFunction(const Func* func, Frame* frame, v
         if ( arg_val >= 0 || addl_val >= 0 )
             arg_events.insert({std::string((*args)[0]->AsString()->CheckString()), make_tuple(arg_val, addl_val)});
     }
-
-    return {true, result};
+    Ref(result.get());
+    return {true, result.get()};
```

Comment:
```suggestion
#if ZEEK_VERSION_NUMBER >= 30200
    return {true, result.release()};
#else
    return {true, result};
#endif
```

Using `release()` is the simpler way to have the caller "adopt a reference-count" since the `IntrusivePtr` is about to destruct and decrement the ref-count anyway.  Then also may want to check `ZEEK_VERSION_NUMBER` to remain compatible with older Zeek versions.